zebra-rpc/src/methods.rs

```diff
@@ -1376,10 +1376,13 @@ fn get_height_from_int(index: i32, tip_height: Height) -> Result<Height> {
         }
         Ok(Height(height))
     } else {
-        let height = (tip_height.0 as i32).checked_add(index + 1);
```

Comment:
I try to avoid the `as` operator in production code, because it casts the bits without checking for overflow:
https://doc.rust-lang.org/stable/reference/expressions/operator-expr.html#numeric-cast

Reply:
Yea, i generally avoid it as well but it seems i missed it. Thanks.